opencv: build failure if Eigen is not installed in osx

In scripts/apothecary:

```
$ apothecary update ofxOpenCv
...

[ 93%] Building CXX object modules/contrib/CMakeFiles/opencv_contrib.dir/src/retinafilter.cpp.o
[ 93%] Building CXX object modules/contrib/CMakeFiles/opencv_contrib.dir/src/rgbdodometry.cpp.o
/Users/ryuyamamoto/cv/openFrameworks/scripts/apothecary/build/opencv/modules/contrib/src/rgbdodometry.cpp:65:12: fatal error: 'unsupported/Eigen/MatrixFunctions' file not found
#  include <unsupported/Eigen/MatrixFunctions>
           ^
1 error generated.
make[2]: *** [modules/contrib/CMakeFiles/opencv_contrib.dir/src/rgbdodometry.cpp.o] Error 1
make[1]: *** [modules/contrib/CMakeFiles/opencv_contrib.dir/all] Error 2
make: *** [all] Error 2
```

I'm definitely sure that adding `-DWITH_EIGEN=OFF` to the cmake fixes this problem. I can create a PR if you are ok to the change.

--- a/addons/ofxOpenCv/scripts/formulas/opencv.sh
+++ b/addons/ofxOpenCv/scripts/formulas/opencv.sh
@@ -96,6 +96,7 @@ function build() {
       -DWITH_V4L=OFF \
       -DWITH_PVAPI=OFF \
       -DWITH_OPENEXR=OFF \
+      -DWITH_EIGEN=OFF \
       -DBUILD_TESTS=OFF \
       -DBUILD_PERF_TESTS=OFF 2>&1 | tee -a ${LOG}
     echo "CMAKE Successful"

